assets/questions/bstClosestValue.js

```diff
@@ -0,0 +1,216 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Find Closest Value in Binary Search Tree.
+ */
+/* eslint no-magic-numbers: ["error", {ignore: [1,4,14.7,2.5,24]}]*/
+globalData.questions['bstClosestValue'] = {  // eslint-disable-line dot-notation
+  title: 'Find Closest Value in Binary Search Tree',
+  starterCode: {
+    python:
+`def findClosestValue(input):
+    root = input[0]
+    target = input[1]
+
+    #TODO : write your code after here
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+        @classmethod
+        def deserialize(cls, data):
+            class TreeNode(object):
+                def __init__(self, x):
+                    self.val = x
+                    self.left = None
+                    self.right = None
+
+
+                def __str__(self):
+                    return self.val
+
+            def constructTree():
+                val = next(vals)
+                if val == '#':
+                    return None
+                node = TreeNode(float(val))
+                node.left = constructTree()
+                node.right = constructTree()
+                return node
+
+            inputs = data.split('|')
+            vals = iter(inputs[0].split())
+            return constructTree(), float(inputs[1])
+
+        @classmethod
+        def failToUseAbsoluteDiff(cls, input):
+            root = input[0]
+            target = input[1]
+
+            kid = root.left if target < root.val else root.right
```

Comment:
could you use a better name than kid? It's not clear what that's supposed to be, here.